rero_ils/config.py

```diff
@@ -2274,7 +2265,7 @@ def _(x):
             )
         ),
         filters={
-            _('request_status'): and_term_filter('status'),
+            _('status'): and_term_filter('status'),
```

Comment:
This should not be modified. It was just fixed, the name of the facet should be 'request_status'

Reply:
Not resolved.